packages/webapp/src/components/PolicyAndCookieBanner/PolicyAndCookieBanner.tsx

```diff
@@ -0,0 +1,123 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { Link, Text } from '@fluentui/react'
+import type { CSSProperties, FC } from 'react'
+import React, { memo, useMemo } from 'react'
+
+import { useLoadMSFTCookieScript } from './PolicyAndCookieBanner.hooks.js'
+import type {
+	CookieConsent,
+	CookieConsentBannerThemes,
+} from './PolicyAndCookieBanner.types.js'
+
+const containerStyles: CSSProperties = {
+	display: 'flex',
+	justifyContent: 'center',
+	alignItems: 'center',
+	gap: '20px',
+	padding: '10px',
+}
+
+export type PolicyLinkDetails = { name: string; href: string }
```

Comment:
Perhaps the prop types could me moved to `PolicyAndCookieBanner.types.ts`?

Reply:
Done!